CMakeLists.txt

```diff
@@ -1,8 +1,15 @@
 cmake_minimum_required(VERSION 3.13)
 
+# Verify that your CMake version is exactly 3.13.x or newer
+if(${CMAKE_VERSION} VERSION_LESS "3.13")
```

Comment:
I think we could remove this now since we have `cmake_minimum_required(VERSION 3.13)`

Reply:
Good point. Removed duplicate check.